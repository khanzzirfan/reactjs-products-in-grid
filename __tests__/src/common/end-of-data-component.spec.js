import React from 'react';
import { shallow, mount } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import EndOfDataComponent from 'src/common/end-of-data-component';

describe("end of data component test", () => {

    let wrapper = null;
    beforeEach(() => {
        wrapper = shallow(<EndOfDataComponent />);
    })

    it("component should be defined", () => {
        expect(wrapper).toBeDefined();
    });

    it("component should match to snapshot", () => {
        expect(wrapper).toMatchSnapshot();
    });

    it("component should render text correctly", () => {
        expect(wrapper.text()).toEqual("~ end of catalogue ~");
    });

    it("component should render classes correctly", () => {
        expect(wrapper.find('.col-md-12 .text-center').exists()).toBeTruthy();
    });

});