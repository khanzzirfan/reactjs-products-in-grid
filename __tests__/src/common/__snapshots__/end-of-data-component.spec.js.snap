// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`end of data component test component should match to snapshot 1`] = `
<div
  className="col-md-12 text-center"
>
  <span>
    ~ end of catalogue ~
  </span>
</div>
`;
